packages/web-client/tests/integration/components/image-editor-test.ts

```diff
@@ -65,6 +82,36 @@ module('Integration | Component | image-editor', function (hooks) {
 
     await click(SAVE_BUTTON);
 
-    assert.equal(result, rotatedAndCroppedImageDataUri);
+    // not sure why settled isn't picking up the async calls?
```

Comment:
It might be because Ember is unaware of the functions, a [test waiter](https://github.com/emberjs/ember-test-waiters) might be of use to register the asynchronous calls so the testing infrastructure knows to wait.